test/db/archos/linux-x64/dbg_dmhd

```diff
@@ -0,0 +1,40 @@
+NAME=dmhd with no allocated bins
```

Comment:
By the way, if you want to add all the various tests you are using to test your code here, or the one suggested in the comments, feel free to add them here and maybe mark them as BROKEN=1 for now.